app/assets/stylesheets/_lists.scss

```diff
@@ -1,3 +1,21 @@
+%default-sub-sublist{
+    ol{
+      list-style-type: lower-roman;
+    }
+    ul{
+      list-style-type: square;
+    }
+}
+%default-sublist{
+  ol{
+    list-style-type: lower-alpha;
+    @extend %default-sub-sublist;
+  }
+  ul{
+    list-style-type: circle;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets